addons/osio-addon/src/test/java/io/fabric8/launcher/osio/steps/OpenShiftStepsTest.java

```diff
@@ -40,7 +40,7 @@
 public class OpenShiftStepsTest {
 
     private static final HoverflyRule HOVERFLY_RULE = createHoverflyProxy("openshiftsteps-simulation.json",
-                                                                          "github.com|githubusercontent.com|api.openshift.io|api.prod-preview.openshift.io|openshiftapps.com");
+                                                                          "github.com|githubusercontent.com|api.openshift.io|prod-preview.openshift.io|openshiftapps.com");
 
     @ClassRule
     public static RuleChain ruleChain = RuleChain
```

Comment:
Can you change the `LAUNCHER_MISSIONCONTROL_OPENSHIFT_API_URL` default value here as well?

Reply:
Done in 0b2f8a5